src/workflows/airqo_etl_utils/satellilte_utils.py

```
@@ -0,0 +1,160 @@
+            "pm2_5",
+            "ID",
+            "date",
+        }
+        if not required_columns.issubset(data.columns):
+            missing_columns = required_columns.difference(data.columns)
+            raise ValueError(
+                f"Provided dataframe missing necessary columns: {', '.join(missing_columns)}"
+            )
+        try:
+            data["date"] = pd.to_datetime(data["date"])
+        except ValueError as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)